editor/editor_node.cpp

```diff
@@ -1225,6 +1225,8 @@ void EditorNode::_save_scene(String p_file, int idx) {
 	}
 	memdelete(dummy_scene);
 
+	scene_tree_dock->get_tree_editor()->update_tree();
+
```

Comment:
Is there a way to make it so that 
- OnChangeInEditorDiscription
  - Check String if(!NullOrEmpty)
    - scene_tree_dock->get_tree_editor()